src/kOS.Safe/Encapsulation/ISerializableValue.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace kOS.Safe.Encapsulation
+{
+    public interface ISerializableValue
```

Comment:
this could be done with the https://msdn.microsoft.com/en-us/library/system.serializableattribute(v=vs.110).aspx class attribute